ee/modules/650-runtime-audit-engine/images/falco/werf.inc.yaml

```
@@ -1,55 +1,46 @@
-{{- $falcoVersion := "0.39.2" }}
-{{- $falcoctlVersion := "0.10.1" }}
-{{- $pluginsVersion := "plugins/kafka/v0.1.3" }}
+# We use https://github.com/falcosecurity/falco/pull/3602 to fix build problems in the Release mode.
+# When this PR will be merged, we will return to the release version of Falco.
+{{- $falcoVersion := "0.41.0" }}
```

```suggestion
{{- $falcoVersion := "0.41.1" }}
```

the new version exists ;) with some bug fixes